sdk/keyvault/azure-security-keyvault-jca/src/main/resources/META-INF/services/java.security.Provider

```diff
@@ -0,0 +1 @@
+com.azure.security.keyvault.jca.KeyVaultJcaProvider
```

Comment:
I am wondering does it work before, since this file is missing in previous versions.